test/pylib/manager_client.py

```diff
@@ -28,11 +28,14 @@
 import aiohttp
 import asyncio
 
+import universalasync
+
 
 logger = logging.getLogger(__name__)
 
 
-class ManagerClient():
+@universalasync.wrap
```

Comment:
I think I found the anti-async trickery :-)
I have no idea what this does. I wonder if the developers who made test/pylib async are fine with this approach. CC @kbr-scylla